common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_common.yaml

```diff
@@ -826,4 +826,101 @@ sSteeringEvSnr:
+    _type: uint8_t
+    _value: 0
+    _comment: size of provided channel_pool
+ channel_pool:
+    _type: uint8_t
+    _length: [ "beerocks::message::SUPPORTED_CHANNELS_LENGTH" ]
+    _comment: pool of channels to be scaned
+
+sChannelScanRequestParams:
+ _type: struct
+ dwell_time_ms:
+    _type: int32_t
+    _value: -1
+    _comment: an invalid (-1) value indicates this value is not requested
+ interval_time_sec:
+    _type: int32_t
+    _value: -1
+    _comment: an invalid (-1) value indicates this value is not requested
+ channel_pool_size:
+    _type: int8_t
+    _value: 0
+    _comment: an invalid (-1) value indicates this value is not requested
+ channel_pool:
+    _type: uint8_t 
+    _length: [ "beerocks::message::SUPPORTED_CHANNELS_LENGTH" ]
+
+sChannelScanResults:
+  _type: struct
+  ssid:
+    _type: char
+    _length: [ "beerocks::message::WIFI_SSID_MAX_LENGTH" ]
+    _comment: The current service set identifier in use by the neighboring WiFi SSID. The value MAY be empty for hidden SSIDs.
+  bssid:
+    _type: sMacAddr
+    _comment: The BSSID used for the neighboring WiFi SSID.
+  mode:
+    _type: char
+    _length: [ "beerocks::message::WIFI_GENERIC_STRING_LENGTH" ]
+    _comment: The mode the neighboring WiFi radio is operating in. Enumeration of AdHoc, Infrastructure
+  channel:
+    _type: uint32_t
+    _comment: The current radio channel used by the neighboring WiFi radio.
+  signal_strength_dBm:
+    _type: int32_t
+    _comment: An indicator of radio signal strength (RSSI) of the neighboring WiFi radio measured in dBm, as an average of the last 100 packets received.
+  security_mode_enabled:
+    _comment: The type of encryption the neighboring WiFi SSID advertises. Enumeration of None, WPA-WPA2 etc.
+    _type: char
+    _length: ["beerocks::message::WIFI_GENERIC_STRING_LENGTH"]
+  encryption_mode:
+    _comment: Comma-separated list of strings. The type of encryption the neighboring WiFi SSID advertises. Each list item is an enumeration of TKIP, AES
+    _type: char
+    _length: ["beerocks::message::WIFI_GENERIC_STRING_LENGTH"]
+  operating_frequency_band:
+    _comment: Indicates the frequency band at which the radio this SSID instance is operating. Enumeration of 2.4GHz, 5GHz
+    _type: char
+    _length: ["beerocks::message::WIFI_OPERATING_STRING_LENGTH"]
+  supported_standards:
+    _comment: Comma-separated list of strings. List items indicate which IEEE 802.11 standards thisResultinstance can support simultaneously, in the frequency band specified byOperatingFrequencyBand. Each list item is an enumeration of
+    _type: char
+    _length: ["beerocks::message::WIFI_GENERIC_STRING_LENGTH"]
+  operating_standards:
+    _comment: Comma-separated list of strings. Each list item MUST be a member of the list reported by theSupportedStandardsparameter. List items indicate which IEEE 802.11 standard that is detected for thisResult.
+    _type: char
+    _length: ["beerocks::message::WIFI_OPERATING_STRING_LENGTH"]
```

Comment:
Just a reminder to drop all the strings in favor of simpler types...

Reply:
refactored to use enums 👍